extensions/amp-story/amp-story.md

```diff
@@ -221,6 +221,22 @@ Disables the [Live story](#Live-story) functionality.
 
 Used with the live-story attribute. Time interval (in milliseconds) between checks for new content. If no `data-poll-interval` is provided it with default to the 15000 millisecond minimum. A value under 15000 milliseconds is invalid.
 
+##### parallax-fx [optional]
+
+Enables the tilt-controlled [3D Parallax Effect](#parallax-fx) which uses the device's accelerometer to give the illusion of a 3D space to the story's layers. The effect is only available on mobile devices.
```

Comment:
is this only available on mobile? I thought there was some discussion about enabling it for desktop (via cursor position) as well?

I think we should be careful about how we message this if we do plan on making it work for desktop in the future. It may be a good idea to just future proof this and make it something like:

`parallax-fx="accelerometer"`

which could then be changed to support something like "always".

Reply:
This PR enables the desktop effect as well, changing the description to match.